tdrs-frontend/package.json

```diff
@@ -32,12 +34,14 @@
     "redux-thunk": "^2.3.0",
     "seamless-immutable": "^7.1.3",
     "typescript": "^3.9.7",
-    "uswds": "2.9.0"
+    "uswds": "2.9.0",
+    "uuid": "^8.3.2"
```

Comment:
`uuid` is added to create unique `uuid` strings to be sent with Reports payloads to the backend